pkg/virt-handler/container-disk/mount.go

```diff
@@ -609,3 +633,198 @@ func (m *mounter) unmountKernelArtifacts(vmi *v1.VirtualMachineInstance) error {
 
 	return fmt.Errorf("kernel artifacts record wasn't found")
 }
+
+func (m *mounter) getContainerDiskPath(vmi *v1.VirtualMachineInstance, volume *v1.Volume, volumeIndex int) (*safepath.Path, error) {
+	sock, err := m.socketPathGetter(vmi, volumeIndex)
+	if err != nil {
+		return nil, ErrDiskContainerGone
```

Comment:
nit:  The caller of `getContainerDiskPath` or caller of the caller could just check `errors.Is(err, os.ErrNotExist)`